File: Sistema de Control Clínico Veterinario/Sistema_Control_Clinico_Veterinario/logica/Consulta.cs
namespace SistemaVeterinario.logica
{
    public class Consulta
    {
        public int IdConsulta { get; set; }
        public string Motivo { get; set; }
        public string Diagnostico { get; set; }
        public string Gravedad { get; set; }
        public Tratamiento Tratamiento { get; set; }
        public string Receta { get; set; }

        public void RegistrarTratamiento(Tratamiento t) => Tratamiento = t;
        public void GenerarReceta(string indicaciones) => Receta = indicaciones;
    }
}

File: Sistema de Control Clínico Veterinario/bluej/Consulta.java
import java.util.Date;

public class Consulta {
    private int idConsulta;
    private Date fecha;
    private String motivo;
    private String observaciones;
    private Diagnostico diagnostico;
    private Tratamiento tratamiento;
    
    public Consulta(int id, String motivo) {
        this.idConsulta = id;
        this.fecha = new Date();
        this.motivo = motivo;
    }
    
    public void registrarDiagnostico(Diagnostico d) { this.diagnostico = d; }
    public void registrarTratamiento(Tratamiento t) { this.tratamiento = t; }
    public int getIdConsulta() { return idConsulta; }
    public Diagnostico getDiagnostico() { return diagnostico; }
    public Tratamiento getTratamiento() { return tratamiento; }
}